src/gennylib/src/context.cpp

```diff
@@ -58,9 +59,17 @@ WorkloadContext::WorkloadContext(YAML::Node node,
 
     _clientPool = poolFactory.makePool();
 
+    // TODO: Remove hasApmOpts when CDRIVER-2931 is resolved.
+    auto apmOpts = clientOpts.apm_opts();
+    bool hasApmOpts = false;
```

Comment:
You could condense lines 64-67 to just `auto hasApmOpts = bool{clientOpts.apm_opts()}`.